api/api/src/main/scala/uk/ac/wellcome/platform/api/models/DocumentFilter.scala

```diff
@@ -1,14 +1,18 @@
 package uk.ac.wellcome.platform.api.models
 
 import uk.ac.wellcome.models.work.internal.AccessStatus
-
 import java.time.LocalDate
 
+import uk.ac.wellcome.display.models.LocationTypeRequest
```

Comment:
Not totally sure about the `Request` naming... but can't think of anything better.

Reply:
We've used it elsewhere, so copying. I think I came up with it and felt weird at the time. Query?

Reply:
Yeah think I prefer query!